test/functional/apps/discover/_discover.js

```diff
@@ -190,6 +190,7 @@ export default function({ getService, getPageObjects }) {
         await kibanaServer.uiSettings.replace({ 'dateFormat:tz': 'America/Phoenix' });
         await browser.refresh();
         await PageObjects.header.awaitKibanaChrome();
+        await queryBar.setQuery('');
```

Comment:
To prevent a test failure caused by the query input of the previous test.

Reply:
no that this just a little fix, those tests should work isolated, but that's a different cup of tea/issue